5 New disadvantages

KRdaMystic opened the pull request:
Serpents Delight, Enduring Angels, Weakened Apothecarion, Small Reclusiam, Barren Librarius

EttyKitty committed:
ci: Fix the auto-compile tag target?

EttyKitty committed:
feat: Copy version number by clicking on it (#184)

Not much to say. Click > play click sound > copy.
The format is the same as displayed in the main menu.

OH296 committed:
fix: Array error with custom select in draw_manage_unit script (#187)

as stated fixes this bug report :
https://discord.com/channels/714022226810372107/1315214830147338281

OH296 committed:
fix: Add extra check to stop array errror in ui_manage ship area (#188)

as on tin hopefully fixes:
https://discord.com/channels/714022226810372107/1315171741932322816

EttyKitty committed:
ci: Change auto-build branch to release/0.9

VanWeapon committed:
docs: Update code_guidance.md (#201)

Mostly formatting, added point_and_click, will add more over time

KRdaMystic committed:
Cherry Picking adv_6

EttyKitty committed:
ci: Update target commit for auto-build

EttyKitty committed:
Revert "ci: Update target commit for auto-build"

This reverts commit f16eb1c8a109ee3003a1cae0649145791b241e06.

EttyKitty committed:
New chapter traits

OH296 committed:
fix: obj_ini.icon string failsafe (#212)

fixes:
https://discord.com/channels/714022226810372107/1318249302799421450

OH296 committed:
fix: Wrong variable name in planet defence battles (#216)

as stated fixes this
https://discord.com/channels/714022226810372107/1312893103434109029

OH296 committed:
fix: Add error handling and split crusade into own function (#217)

hopefully either solves or provides better error handling for this:
https://discord.com/channels/714022226810372107/1312791321068572793

OH296 committed:
fix: Return trade ships home or delete them (#218)

trade ships will now hopefully return home but if no valid location will just destroy and stop cluttering the map.
https://discord.com/channels/714022226810372107/1299419464907100280

OH296 committed:
fix: Add checks to stop empty strings in item logic (#213)

as stated fixes this
https://discord.com/channels/714022226810372107/1318242260244238457

OH296 committed:
fix: Fix a bunch of ship errors caused by location (#214)

more 0 to -1 lid variable conversions.

as stated in title fixes:
https://discord.com/channels/714022226810372107/1318240888534601758 and
this https://discord.com/channels/714022226810372107/1318242740605030461

OH296 committed:
fix: Unset global variable crashing artificer armour (#215)

add error handling to drawing unit imagine.
as stated should fix:
https://discord.com/channels/714022226810372107/1318224998179012669

EttyKitty committed:
Merge branch 'compile/main' into release/0.10.0.0

EttyKitty committed:
fix: Chain Axe > Chainaxe

EttyKitty committed:
ci: Double nightly tag test

90% chance it'll not work.

KRdaMystic committed:
New Disadvs

2 disadvs suggested by "Themigratingcoconut"
3 minor disadvs to subtract specs

OH296 committed:
fix: Crash at the end of scr_purge_world from unset variable (#223)

OH296 committed:
feat: Deselect dreadnoughts on attack screen and ship load fix (#222)

- adds button on drop select to control if player wishes dreadnoughts in battle or not
- fixes previous bug with being unable to load marines onto ships

EttyKitty committed:
ci: Fix wrong step name

OH296 commented on objects/obj_creation/Create_0.gml:
```diff
@@ -806,20 +806,20 @@ var all_disadvantages = [
     {
         name : "Small Reclusiam",
-        description : "Your chapter has a smaller reclusiam compared to other chapters.",
+        description : "Your chapter cares little for its reclusiam compared to other chapters fewer marines have shown the desire to be chaplains.",
```
do you want me to go over the trait distribution mechanics as it would seem like a good idea to maybe alter some of the trait distributions to reflect these traits? Either that or i can just twiddle em but i feel like i've probably done a lot of that and may be slightly biased in my choices at this point

KRdaMystic replied:
If its anything like how melee enthusiasts or Slow and purposeful worked I can figure it out.

OH296 committed:
fix: Crash that could occur when saving the chapter icon path (#224)

solves this:
https://discord.com/channels/714022226810372107/1318805756027469895

OH296 committed:
feat: Add Terminator bionics sprites (#226)

- new terminator arm sprite
- new terminator leg sprite
- new terminator eye sprite
- all terminator bionics drawing switched over to new component drawing system

OH296 committed:
fix: Align warp lanes with stars after game restart (#225)

Fixes a bug where warp lanes would not line up with stars after restarting a game.

EttyKitty committed:
chore: Minor edits to error reports (#228)

- Separate commit history link and commit hash.
- Separated the error message details with newlines for better readability.
- `markdown_codeblock` function supports code language prefix argument.

EttyKitty committed:
ci: Fix nightly building

EttyKitty committed:
docs: Add discord badge to readme (#233)

- Add a Discord badge to the README.
- Fix a typo.

Co-authored-by: sourcery-ai[bot] <58596630+sourcery-ai[bot]@users.noreply.github.com>

EttyKitty committed:
docs: Add discord badge to readme (#233)

- Add a Discord badge to the README.
- Fix a typo.

Co-authored-by: sourcery-ai[bot] <58596630+sourcery-ai[bot]@users.noreply.github.com>

EttyKitty committed:
fix: "Unstretch" the main menu background (#236)

OH296 committed:
fix: Uncommon crash from squad temporary variable (#230)

- Fixed a crash related to the "squad" temporary variable not being correctly managed.

OH296 committed:
fix: Unloading not possible directly after loading marines (#229)

- temp variables being set incorrectly upon loading a marine
- fixes https://discord.com/channels/714022226810372107/1318240888534601758

OH296 committed:
feat: Improve missing ships return from warp (#239)

- Ships returning from warp can now be captured by a Chaos fleet.
- Ships returning from warp can now be destroyed, scattering any surviving marines in the system.

OH296 committed:
fix: Array error with image caching (#240)

- Fix an array out-of-bounds error that occurred when caching images with an ID larger than the current size of the cache.

OH296 committed:
fix: Ruins one last time (#237)

Fix crashes related to ruins battles and player fleet arrays.

OH296 committed:
fix: Modernise and remove redundant code scr_random_event (#242)

- Fix a bug where the script was iterating from 1 to 100 instead of iterating over the length of the event array.

OH296 committed:
fix: Several bugs with fleet combat (#231)

- Clear vehicle variables when resetting vehicle arrays.
- Fixed a crash caused by incorrect referencing of the "luck" variable.
- Fixed a crash caused by ships not having a max speed variable.
- Improved error handling in ship cleanup scripts.

EttyKitty committed:
fix: Terminator bionic arms errors (#234)

- Fix a [crash](https://discord.com/channels/714022226810372107/1320107379798577183).
- Refactor and format arm drawing code.
- Cleanup garbage files.
- Account for some edge cases.

EttyKitty committed:
fix: Mobility gear on Terminators (#235)

- Fixed a bug that prevented terminators from equipping mobility gear.

OH296 committed:
fix: Array error in ship self-healing crash (#243)

- Fix an array error that caused a crash during the self-healing process for ships.

EttyKitty committed:
fix: Research progression and add research complete popup (#244)

Fix bug in manufacturing research progression and add a popup to notify the player when research is complete.

OH296 committed:
fix: Tech heretic conversion logic (#241)

- Fix an issue where the tech heresy conversion logic could cause errors.

EttyKitty committed:
refactor: Move experience reward logic to a script (#245)

- Moved exp reward logic into a separate script, to fix broken rewards.
- Fixed some typos.

OH296 committed:
fix: Loading forge queue (#246)

- Fix an issue where the forge queue was not loading correctly.

OH296 committed:
improve error handling, log repairs

OH296 committed:
forge queue visual error with table

OH296 committed:
comparing arrays fix

EttyKitty committed:
ci: Test clone releases action

EttyKitty committed:
ci: Test clone releases action

OH296 committed:
fix : Solve visual errors in forge (#322)

- solves visual error with forge queue with manufactoring research
- add more error handling

## Summary by Sourcery

Bug Fixes:
- Fix visual overlap with research projects in the forge queue display.

OH296 committed:
fix : draw unit image comparing arrays not possible  (#323)

## Summary by Sourcery

Bug Fixes:
- Fix the condition to check for non-empty bionic options array using
array_length instead of direct comparison.

OH296 committed:
trying to round a string

OH296 committed:
PR: Fix trying to round a string in ui_manage (#324)

## Summary by Sourcery

Bug Fixes:
- Prevent error when displaying experience value if it is not a number.

OH296 committed:
move specialist point handler intialisation point

EttyKitty committed:
ci: Update access key name

EttyKitty committed:
ci: Update access key name

OH296 committed:
partial solve to icon crash issue

OH296 committed:
PR: Fix Initialisation sequences in load script causing crashes  (#326)

## Summary by Sourcery

Bug Fixes:
- Prevent crashes caused by missing chapter icon sprites during loading.

OH296 committed:
role not initialised correctly in add man

OH296 committed:
PR: Fix role not being initialised correctly in scr_add_man (#327)

## Summary by Sourcery

Bug Fixes:
- Fixed a bug where the unit's role was not initialised correctly when
adding a new man.

OH296 committed:
fix: Use of safer string interpolation (#328)

- Prevent potential bugs when deleting save files by using string interpolation instead of string concatenation.

EttyKitty committed:
fix: A couple of scr_image_cache errors (#330)

- Fix loading custom chapter images by handling cases where the image path is not set.
- Fix the wrong array push target.
- Format the function.

EttyKitty committed:
ci: Try write-all premission to fix tag move

EttyKitty committed:
ci: Replace write-all premission with a WORKFLOW_TOKEN

At this point, I'm out of ideas on how to fix this.

OH296 committed:
custom work arounds

OH296 committed:
PR: Fix Custom Livery for heads to match departments (#334)

as reported here
https://discord.com/channels/714022226810372107/1317251464242135132/1321969505575895040
custom livery for department heads is wrong
## Summary by Sourcery

Bug Fixes:
- Fixed an issue where heads did not have the correct livery.

OH296 committed:
PR: Fix Fleet retreating crash (#335)

- fixes crash with fleet retreating
- solves a crash with reallocation of marines on ships
- solves minor error in en_fleet alarm 1 from wrong scoping

## Summary by Sourcery

Implement fleet retreating logic and fix a crash caused by reallocating
marines on ships.

Bug Fixes:
- Fix a crash that occurred when a fleet retreated.
- Fix an issue where marines were not correctly reallocated when ships
were destroyed during a retreat.

Enhancements:
- Implement a new `player_retreat_from_fleet_combat` function to handle
fleet retreat logic.

OH296 committed:
PR: Fix minor scoping error in one of the new ship return events (#336)

## Summary by Sourcery

Bug Fixes:
- Fix an issue where piety was not correctly used when calculating
corruption for returned ships.

OH296 committed:
PR: Fix bug from later Initialisation of local complex_set variable moved in draw_unit_image (#338)

## Summary by Sourcery

Bug Fixes:
- Move the initialization of the local variable 'complex_set' to the
beginning of the 'scr_draw_unit_image' function to ensure it is properly
initialized before use.

EttyKitty committed:
docs: Test addition of a PR template (#329)

- Add a pull request template with sections for description, reasons, and related links.

EttyKitty committed:
docs: Test addition of a PR template (#329)

- Add a pull request template with sections for description, reasons, and related links.

OH296 committed:
PR : Fix possible over running of array length with ship weapons (#337)

## Summary by Sourcery

Bug Fixes:
- Prevent crashes caused by mismatched array lengths when accessing ship
weapons and their facing directions.

---------

Co-authored-by: sourcery-ai[bot] <58596630+sourcery-ai[bot]@users.noreply.github.com>

EttyKitty committed:
ci: Add PR title validation workflow (#341)

- Validate pull request titles using the `thehanimo/pr-title-checker` GitHub Action.

EttyKitty committed:
ci: Add PR title validation workflow (#341)

- Validate pull request titles using the `thehanimo/pr-title-checker` GitHub Action.

OH296 committed:
feat(balance): Tweaks to grav weapon as per play testing (#339)

Increase the damage of the Grav-Pistol and add maintenance values for grav weapons.

OH296 committed:
PR: Stop bug where Squad View no longer appears (#340)

- attempts to fix the common bug of the squad view becoming inaccessible
- it appears it was due to a clause making it non viewable if instances
of obj_temp3 existed
- attempt to remove as many uneeded obj_temp3 references as possible
that can be done safely and easily

## Summary by Sourcery

Bug Fixes:
- Fix a bug that prevented the Squad View from appearing when instances
of obj_temp3 existed.

VanWeapon committed:
fix: Custom role saving (#345)

- Fixes custom roles not saving during custom chapter creation.

VanWeapon committed:
fix: Homeworld save/load for custom chapters (#347)

- Fixes an issue where all loaded custom chapters had a hive world due to a typo in scr_save_chapter.

OH296 committed:
Sprite: New Complex Components for Tartaros Armour (#342)

- Tartaros right and left leg components
- Tartaros chest components
- new complex area key "gorget"
- Tartaros gorget components

I'm no artist this is as good as i can do i'm afraid

KRdaMystic committed:
PR: Create Speacial Traits for first founding Chapters (#344)

<!--- You can add `@sourcery-ai` into the title, so that the bot
auto-generates a title -->
## Description of changes
-Adds starting traits for marines in founding chapters. 
## Reasons for changes
-Add more variance between the 9 founding chapters.
## Related links
-The
[project](https://discord.com/channels/714022226810372107/1320186099641090128)

## Summary by Sourcery

New Features:
- Introduced unique starting traits for each of the nine founding
chapters of Space Marines to increase variance and diversity between
them.

OH296 committed:
fix: Artifact continuity issue (#348)

- solves various issues with artifact continuity.
- artifacts were not being destroyed when not being recovered off of
killed marines leaving them in limbo
- occasionally the artifact in question being de-equipped due to being
gifted or some other actor was not equipped to the array loc of the
stated unit
- to solve this the de-equip function now searches for amarine with the
artifact equipped and manually de-equips it from them

## Summary by Sourcery

Fix artifact continuity issues by ensuring artifacts are correctly
de-equipped from units and destroyed when no longer recoverable.

Bug Fixes:
- Resolved issues with artifacts not being destroyed when not recovered
from killed marines.
- Fixed a bug where de-equipping an artifact due to gifting or other
actions did not correctly remove it from the unit\'s inventory.

OH296 committed:
PR : Fix drop_select not displaying in screen correctly (#349)

fixes -
https://discord.com/channels/714022226810372107/1322615907259908158

## Summary by Sourcery

Bug Fixes:
- Fix incorrect centering of the drop-down menu.

OH296 committed:
fix: Uncommon variable type error in ui_manage (#350)

https://discord.com/channels/714022226810372107/1322620181586907160
fixes this
basically just a minor type error

## Summary by Sourcery

Bug Fixes:
- Fix a type error that occurred when `selection_data` did not have a
`purpose` field.

EttyKitty committed:
fix: Captains and CM not healing (loop from 1) (#351)

Changed the loop start index from 1 to 0 in the apothecary_simple function to include all units in the healing check.

KRdaMystic committed:
PR : Adjust founding chapter trait values (#353)

## Description of changes
- Adjusts founding chapter trait values
## Reasons for changes
- Old values resulted in negative effects

## Summary by Sourcery

Enhancements:
- Updated trait values for several founding chapters.

OH296 committed:
PR: New Sprites -  Tartaros face plate, helm and trim variants (#354)

## Summary by Sourcery

New Features:
- Added Tartaros faceplate, helm, and trim variants.

---------

Co-authored-by: EttyKitty <20323032+EttyKitty@users.noreply.github.com>

OH296 committed:
PR : Improve ordering and reduce duplicate line scr_add_man (#355)

fixes-
https://discord.com/channels/714022226810372107/1120687959365128243/threads/1322910920816721980

## Summary by Sourcery

Bug Fixes:
- Fix the order of operations when adding experience and updating the
role of a new marine.

EttyKitty committed:
PR: Fix And improve the advisor text on the recruitment screen (#343)

## Summary by Sourcery

Improve the advisor text regarding recruitment and gene-seed status.

Bug Fixes:
- Fix multiple grammar and style issues in the advisor text.
- Correct the logic for displaying the recruitment status based on the
number of marines and the recruiting status.
- Fix the logic for displaying the gene-seed status and the number of
recruiting worlds.

VanWeapon committed:
fix: Boarder sprites and fleet sprites being wrong (#358)

- Correct the display of boarder and fleet sprites by adjusting the image indexing in `scr_image` calls.

OH296 committed:
PR: Fix spawning bugs with outdated function name (#359)

- bug from outdated function name in scr_add_man
- occasionally stored unit.ship_location would go stale 
- will need to find better solution in future in theory scr_ship_clean
should prevent this

## Summary by Sourcery

Bug Fixes:
- Fixed a bug that caused marine location data to become stale.

OH296 committed:
feat(sprite): Mk8 components (#356)

- add gorget component to mk8 and make some variants
- remove gorget from base mk8 so that more mk7 helms are compatible to
be drawn with mk8
- add mk8 chest components that can extend variation with mk7 chest
components

## Summary by Sourcery

New Features:
- Added new gorget and chest pieces for the Mk 8 armour set.

---------

Co-authored-by: EttyKitty <20323032+EttyKitty@users.noreply.github.com>

OH296 committed:
fix: Vehicle purchase (#360)

- brings the scr_add_vehicle script in line with the changes to variable
array lengths with ships

## Summary by Sourcery

Bug Fixes:
- Fix an issue with adding vehicles to companies with variable-length
ship arrays.

OH296 committed:
PR: Fix forge assignment bugs (#361)

- general overhaul of FeatureSelected struct
- correct defualt options for <unit>.is_at_location
- correct tracking of techmarine forge assignments fixed

## Summary by Sourcery

Fix forge assignment bugs and improve the FeatureSelected struct.

Bug Fixes:
- Correct default options for unit location checks.
- Correct tracking of Techmarine forge assignments.

Enhancements:
- Overhaul of the FeatureSelected struct to improve data handling.

KRdaMystic committed:
PR: Improve Balance Obliterated trait (more specialists)(balance) (#364)

gives obliterated a fighting chance.

<!--- PR title format should be `commit type: Title` -->
<!--- Commit types can be found at
https://github.com/pvdlg/conventional-commit-types?tab=readme-ov-file#commit-types
-->
<!--- You can add `@sourcery-ai` into the title, so that the bot
auto-generates a title -->
## Description of changes
<!--- Leverage the list functionality, if you have many changes -->
-Adds one specialist to each of the offices to avoid inquisition
penalties
-Strength points over 5 now count for 10 scouts in place of 5
-removed the `siege = 0` from obliterated's block of code
## Reasons for changes
<!--- Leverage the list functionality, here as well -->
- Making obliterated more playable instead of the current guaranteed
traitor declaration by the inquisitor.

## Summary by Sourcery

Rebalance the "Obliterated" trait by reducing penalties to specialist
counts and increasing the scout bonus for strength points above 5.

Bug Fixes:
- Reduce the penalties applied to specialist counts for the
"Obliterated" trait.

Enhancements:
- Increase the scout bonus from strength points over 5 for "Obliterated"
trait.

KRdaMystic committed:
PR:Allow Tactical Bikers and Marines to sabotage  when chapter are ambushers(#365)

## Description of changes
<!--- Leverage the list functionality, if you have many changes -->
-Allows Tactical Bikers to do sabotage missions
-Allows Tactical marines to do sabotage missions for chapters with
Ambushers
## Reasons for changes
<!--- Leverage the list functionality, here as well -->
-Add more to Lightning warriors and Ambushers to make them more
competetive as advantages.

## Summary by Sourcery

New Features:
- Tactical Bikers and Tactical Marines with the "Ambushers" advantage
can now perform sabotage missions.

VanWeapon committed:
PR:  Player can define extra vehicles in custom chapter json (#362)

<!--- PR title format should be `commit type: Title` -->
<!--- Commit types can be found at
https://github.com/pvdlg/conventional-commit-types?tab=readme-ov-file#commit-types
-->
<!--- You can add `@sourcery-ai` into the title, so that the bot
auto-generates a title -->
## Description of changes
<!--- Leverage the list functionality, if you have many changes -->
- implements the `extra_vehicles` attribute from chapter jsons which
works on init now. All extra vehicles are added to the 10th company so
they show up on the homeworld at the beginning of the game
## Reasons for changes
<!--- Leverage the list functionality, here as well -->
- implement unfinished feature
## Related links
<!--- Other PRs; Discord bug reports, messages, threads, etc.; outside
docs, etc.; -->
-
https://discord.com/channels/714022226810372107/714027491412410369/1323805664723734549

## Summary by Sourcery

New Features:
- Added support for the "extra_vehicles" attribute in chapter JSON files
to add vehicles to the 10th company during game initialization.

OH296 committed:
fix: Repair slaughtersong event (#370)

EttyKitty committed:
ci: Update pull_request_template (#369)

EttyKitty committed:
ci: Update pull_request_template (#369)

EttyKitty committed:
fix(Marine Spawning): Roll armor works on termies again and is improved (#368)

OH296 committed:
fix: ai_a script increase efficiency and error handling to find unresolved crash (#371)

OH296 committed:
fix: Planet and faction sprites displaying wrong sprites (#372)

OH296 committed:
fix: Crash from unset sector name (#373)

OH296 committed:
fix: Incorrect method name causing crash during Spyer fight (#374)

OH296 committed:
fix: improve ship combat checks (#375)

VanWeapon committed:
fix: Tidy up the aspirant trial display on chapter creation slides (#377)

VanWeapon committed:
refactor: Chapter trait checking standardisation  (#376)

EttyKitty committed:
ci: A bunch of minor CI changes (#378)

EttyKitty committed:
ci: A bunch of minor CI changes (#378)

KRdaMystic committed:
fix: Weapon and formation options for Bikers and Assaults (#357)

- Fixes optional weapons for bikers and assaults to utilize weapon lists.
- Fixes formation location selection options for aforementioned squad types

EttyKitty committed:
fix: A couple of missed old trait checks (#379)

EttyKitty committed:
refactor: Wrap a ton of stuff into try/catch and format (#381)

EttyKitty committed:
fix: Variable scope in scr_add_vehicle (#385)

Co-authored-by: Nelsonh <81228864+OH296@users.noreply.github.com>

EttyKitty committed:
fix: Misplaced broken trait check (#382)

Fixed a crash that occurred on game load.

EttyKitty committed:
fix(sprite): Iron Halo display position on Artificer Armor (#383)

EttyKitty committed:
fix: Term Sergeant and Veteran broken loadouts (#388)

EttyKitty committed:
revert: Armour randomization by default (#389)

OH296 committed:
fix: Add safeguards and streamline adding ship to combat (#390)

OH296 committed:
fix: controller alarm_7 use modern ship methods (#391)

OH296 committed:
fix: move to modern method loading unloading troops controller step (#394)

OH296 committed:
fix: error handling to solve issue with sector name (#393)

OH296 committed:
fix: Typo making lava planet sprite not display (#395)

OH296 committed:
fix: Controller alarm 3 (#396)

OH296 committed:
fix: Manage hot keys interrupting company naming  (#397)

OH296 committed:
fix: PlanetData error handling to planet_forces variable (#392)

OH296 committed:
fix: Vehicle spawning (#398)

OH296 committed:
fix: Trying to call alarm on non existing object (#399)

OH296 committed:
fix: default options not being set breaking up squads loadouts (#400)

OH296 committed:
fix: add default argument for adding items (#401)

OH296 committed:
fix: typo in alarm7 obj_ncombat from earlier fix (#405)

OH296 committed:
fix: Player ship class and arrays (#406)

OH296 committed:
fix: random caching error with UnitButtonObject (#407)

OH296 committed:
fix: whelp better give it a go ruins shit (just go with it) (#408)

EttyKitty committed:
Merge branch 'stable-candidate'

VanWeapon committed:
chore: json file error (#366)

EttyKitty committed:
feat: Separate armor heads (#367)

EttyKitty committed:
refactor: First company command gear/size assign (#402)

EttyKitty committed:
feat: Edits to robes, psy hood and daemon binders (#403)

EttyKitty committed:
feat: Remove "Power Weapon" from the player side (#411)

OH296 committed:
fix: Repair old guard armour distribution system (#413)

OH296 committed:
feat: new skitarii base sprite (#409)

OH296 committed:
feat: Allow land speeders on raids (#386)

OH296 committed:
fix: get rid of restart variables to stop all related crashes (#414)

OH296 committed:
fix: wrong race variable on new recruits (#416)

EttyKitty committed:
fix(visual): Force Sword/Axe side flip (#415)

OH296 committed:
fix: Potential array issue with self-healing ships (#417)

OH296 committed:
fix: Redundant check causing an early game over (#418)

OH296 committed:
fix: odd bug from player being wiped out in battle (#419)

OH296 committed:
fix: check inquisitor in array range (#428)

OH296 committed:
fix: Use correct index for missions (#427)

OH296 committed:
fix: case typo on shrine for recruit data (#429)

OH296 committed:
fix: Wrong argument passed into system_feature_bool (#430)

OH296 committed:
fix: ai_a error improve exception message (#432)

OH296 committed:
feat: Refactor bionic legs drawing and add variant (#404)

EttyKitty committed:
feat: Wolf-Priest helmet; Chaplain helm drawing refactor (#433)

EttyKitty committed:
feat: Helmets adjustments (#434)

VanWeapon committed:
refactor: Diplomacy and advisor icon refactor to new image system (#363)

OH296 committed:
fix: Error handling to unit_finder_panel (#436)

OH296 committed:
refactor: Memory safe drawing of components (#435)

Co-authored-by: EttyKitty <20323032+EttyKitty@users.noreply.github.com>

EttyKitty committed:
ci: More title checker triggers (#437)

OH296 committed:
fix: star_select huge rework with bug fix focus (#438)

OH296 committed:
fix: Typo causes crash when problem on planet (#442)

OH296 committed:
fix: custom_advisor splash images breaking game after save (#443)

EttyKitty committed:
fix: Change error handling of NameGenerator (#439)

OH296 committed:
fix: Move variable declaration to solve crash on drop_Select (#444)

OH296 committed:
fix: add init of custom_advisors in obj_ini creation (#445)

OH296 committed:
fix: typo causing crash when destroying gene pods (#446)

OH296 committed:
feat: Concept shaders complex livery and cloaks  (#440)

Co-authored-by: EttyKitty <20323032+EttyKitty@users.noreply.github.com>

OH296 committed:
feat: New Player system spawn options (#384)

OH296 committed:
fix: clean up a few bugs with new planet options (#447)

OH296 committed:
fix: selection of research method changed to allow ships to be selected (#448)

OH296 committed:
fix: many misc sprite bugs (#449)

VanWeapon committed:
fix: Attempted fix for crash on image id 0 (#450)

VanWeapon committed:
fix: Minor alignment issues with save chapter button (#451)

OH296 committed:
Fix: gloriana vessels work again (#453)

VanWeapon committed:
fix: Diplo splash images for trade dialogue (#454)

OH296 committed:
fix: Colonists and introduce PurchaseButton constructor (#456)

KRdaMystic committed:
Trait bonuses

 adds more maluses to the new disadvs

OH296 committed:
fix: Make dungeon battles and local battles run on modern roster system (#459)

OH296 committed:
fix: Issue wrong  spawn name would spawn for home planet (#460)

OH296 committed:
feat: Blood Angels Complex livery (#461)

VanWeapon committed:
fix: Random exit from creation screen on click (#462)

MCPO-Spartan-117 committed:
fix: `Ambushers`+`Lightning Warriors` crash (#464)

MCPO-Spartan-117 committed:
fix: Homeworld and recruiting world names mixing up (#465)

MCPO-Spartan-117 committed:
fix: Tech Heresy detection var in wrong scope (#466)

MCPO-Spartan-117 committed:
feat: If population equals zero then reset all influence (#468)

MCPO-Spartan-117 committed:
feat: Make Purge interaction with Cults more obvious (#467)

EttyKitty committed:
feat: New sprites for Adeptas (#469)

frost923 committed:
fix: Rhino missing HK missile armament (#472)

EttyKitty committed:
feat: Add pasting support for cheats (#470)

MCPO-Spartan-117 committed:
feat: Make Bombard purging reduce corruption (#473)

EttyKitty committed:
feat: Rework how debugl is used and works (#476)

MCPO-Spartan-117 committed:
fix: Garrison fixes and minor changes (#475)

MCPO-Spartan-117 committed:
fix: Remove tau influence from corruption cals (#479)

MCPO-Spartan-117 committed:
feat: Custom and more event colors (#478)

Co-authored-by: Nelsonh <Humes101@gmail.com>

MCPO-Spartan-117 committed:
fix: Disable annoying events (#481)

MCPO-Spartan-117 committed:
fix: Boarders experience out of range (#485)

MCPO-Spartan-117 committed:
fix: More debug and more fixes for garrisons (#486)

Frost-bit-jpg committed:
fix: Enhanced Item Accessibility and Progression (#463)

Co-authored-by: Nelsonh <81228864+OH296@users.noreply.github.com>

Blogaugis committed:
fix: make techmarine equipment producible in forges (#487)
